javascript_config.php

```diff
@@ -27,9 +27,14 @@
 # Prevent output of HTML in the content if errors occur
 define( 'DISABLE_INLINE_ERROR_REPORTING', true );
 
+# Suppress default headers. This allows cachig as defined in server configuration
+$g_bypass_headers = true;
```

Comment:
Should you skip headers that disable caching only when expected query parameter is supplied to handle the case where plugins include such files without query parameters?